### PR TITLE
PSP migration docs announcement.

### LINKED ISSUE #88: Write a blog post about the PSP migration docs.
Write a blog post to announce the recent added docs about PSP migration

https://docs.kubewarden.io/psp-migration.html

### PR DESCRIPTION
**Pull request by jvanz**
Adds a very short blog post about the recent documentation explaining to user how migrate from PSP to Kuberwarden policies.

Fixes #88

### CODE REVIEW

File: content/blog/2022/05/psp-migration-docs.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Have you already migrate your Kubernetes PodSecurityPolicy?"
```
Comment (divya-mohan0209) [inline content/blog/2022/05/psp-migration-docs.md:2]:
```suggestion
title: "Have you migrated your Kubernetes PodSecurityPolicy?"
```

File: content/blog/2022/05/psp-migration-docs.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+authors:
+- José Guilherme Vanz
+date: 2022-05-16
+---
+
+If you use the deprecated Kubernetes PodSecurityPolicy (a.k.a PSP), you should
```
Comment (divya-mohan0209) [inline content/blog/2022/05/psp-migration-docs.md:8]:
```suggestion
If you use a version of Kubernetes (< v1.24) that supports the deprecated PodSecurityPolicy (a.k.a PSP), you would be
```

File: content/blog/2022/05/psp-migration-docs.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+wonder what to do after the Kubernetes v1.25 version. When the PSP will be removed.
```
Comment (divya-mohan0209) [inline content/blog/2022/05/psp-migration-docs.md:9]:
```suggestion
wondering what to do after the Kubernetes v1.25 version when the PSP will be removed.
```

File: content/blog/2022/05/psp-migration-docs.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+If this in mind, the Kuberwarden team wrote a [documentation](https://docs.kubewarden.io/psp-migration.html)
```
Comment (divya-mohan0209) [inline content/blog/2022/05/psp-migration-docs.md:10]:
```suggestion
With this in mind, the Kuberwarden team wrote a [documentation](https://docs.kubewarden.io/psp-migration.html)
```

File: content/blog/2022/05/psp-migration-docs.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+to help users migrate away from PSPs to Kuberwarden policies.
+
```
Comment (divya-mohan0209) [inline content/blog/2022/05/psp-migration-docs.md:12]: Can we briefly describe what we've mentioned in the docs for a quick overview? The ending seems to be too abrupt

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Slight rephrasing and suggestions for enhancing readability. I hope you find them useful

File: content/blog/2022/05/psp-migration-docs.md (hunk 5 above)
Comment (flavio) [reply content/blog/2022/05/psp-migration-docs.md:12]: Turning on my random text generator:

> As you know, the original Pod Security Policies had many configuration knobs. The Kubewarden team created a series of policies that offer a 100% feature parity with all the soon to be dropped Pod Security Policies.
> This section of our documentation highlights all these policies and guides you through their recommended settings.
>
> On this topic, do you know our helm chart provides a flag that enforces a chosen selection of Pod Security Policies? This makes even easier to keep your clusters safe!

This is just an idea, feel free to change it as you wish

Comment (viccuad) [reply content/blog/2022/05/psp-migration-docs.md:12]: I like that paragraph from Flavio.

Comment (jvanz) [reply content/blog/2022/05/psp-migration-docs.md:12]: I've just added it in the text. ;)

**Review by flavio — APPROVED** [review]
LGTM, let's publish it tomorrow. Today we've already published https://www.kubewarden.io/blog/2022/05/monitor-mode/

**Pull request merged**